mix.exs

```diff
@@ -18,6 +18,7 @@ defmodule PropCheck.Mixfile do
      description: description(),
      propcheck: [counter_examples: "_build/propcheck.ctx"],
      aliases: aliases(),
+     preferred_cli_env: [tests: :test],
```

Comment:
When was `preferred_cli_env` added to mix? To be honest, I see it here for the first time. Does this require a dependency to a higher Elixir version?

Reply:
It was added in 1.3.0, PropCheck needs 1.5 so no version change needed.

Reply:
TIL: I should read the the beautiful manual more properly